packages/react-storage/src/components/StorageBrowser/components/DataTable.tsx

```diff
@@ -16,6 +16,7 @@ export const TABLE_HEADER_CLASS_NAME = `${TABLE_CLASS_NAME}__header`;
 export const TABLE_HEADER_BUTTON_CLASS_NAME = `${TABLE_CLASS_NAME}__header__button`;
 export const TABLE_ROW_CLASS_NAME = `${TABLE_CLASS_NAME}__row`;
 export const TABLE_DATA_CLASS_NAME = `${TABLE_CLASS_NAME}__data`;
+export const TABLE_DATA_TEXT_CLASS_NAME = `${TABLE_CLASS_NAME}__data__text`;
```

Comment:
Can we use the existing `TABLE_DATA_CLASS_NAME ` instead of adding an additional class name?

Reply:
HMM let me see if I can drop the span. I think it makes sense if we keep this span element to have different classes on them since they are useful hooks for developers to style.

But, I'll see if we can forego the span element all together. We still need something to truncate text and make the items line up so they don't look goofy when in a table (like the LocationsDetailView) where some of the name columns have buttons, and some have padding.

Reply:
Think your approach make sense actually, didn't realize `TABLE_DATA_CLASS_NAME` was being applied to the wrapping `TableDataElement` already